spec/integration/data_stream/_dev/test/system/config.spec.yml

```diff
@@ -23,3 +23,125 @@ spec:
       items:
         type: string
         example: "error.message"
+    agent:
+      description: "Configuration overrides for the Elastic Agent"
+      type: object
+      additionalProperties: false
```

Comment:
Would it be better to let define new properties here to allow `elastic-package` add new settings easily? @jsoriano

Reply:
Yes, I think we can be less restrictive here than in other places.